sympy/polys/matrices/domainmatrix.py

```diff
@@ -57,13 +88,61 @@ def get_domain(cls, items_sympy, **kwargs):
         return K, items_K
 
     def convert_to(self, K):
+        r"""
+            convert_to: Returns a DomainMatrix with the desired domain or field
+
+            >>> from sympy import Matrix, CC
+            >>> from sympy.polys.matrices import DomainMatrix
+            >>> matrix = Matrix([[99, 16, 30, 62], [65, 79, 94, 15], [38, 13, 31, 47], [17, 2, 53, 78]])
+            >>> domainMatrix = DomainMatrix.from_Matrix(matrix)
+            >>> domainMatrix
+            DomainMatrix([[99, 16, 30, 62], [65, 79, 94, 15], [38, 13, 31, 47], [17, 2, 53, 78]], (4, 4), ZZ)
+
+            >>> domainMatrix.convert_to(CC)
+            DomainMatrix([[(99.0 + 0.0j), (16.0 + 0.0j), (30.0 + 0.0j), (62.0 + 0.0j)], [(65.0 + 0.0j), (79.0 + 0.0j), (94.0 + 0.0j), (15.0 + 0.0j)],
+            [(38.0 + 0.0j), (13.0 + 0.0j), (31.0 + 0.0j), (47.0 + 0.0j)], [(17.0 + 0.0j), (2.0 + 0.0j), (53.0 + 0.0j), (78.0 + 0.0j)]], (4, 4), CC)
+
+        """
         return self.from_rep(self.rep.convert_to(K))
 
     def to_field(self):
+        r"""
+            to_field: Returns a DomainMatrix with the appropriate field
```

Comment:
You don't need to repeat the method name. The one line summary should be on the first line of the docstring.

Reply:
Sure.
Thanks.